programs/eosio-launcher/main.cpp

```diff
@@ -1151,11 +1159,11 @@ launcher_def::write_config_file (tn_node_def &node) {
     for (auto &p : node.producers) {
       cfg << "producer-name = " << p << "\n";
     }
-    cfg << "plugin = eosio::producer_plugin\n";
+    cfg << "plugin = " << producer_plugin_str << "\n";
   }
-  cfg << "plugin = eosio::net_plugin\n";
-  cfg << "plugin = eosio::chain_api_plugin\n"
-      << "plugin = eosio::history_api_plugin\n";
+  cfg << "plugin = " << net_plugin_str << "\n";
+  cfg << "plugin = " << chain_api_plugin_str << "\n"
+      << "plugin = " << history_api_plugin_str << "\n";
```

Comment:
I am not sure if EPE 184 just mean to replace a string literal with a variable.

Reply:
What's your idea then?